src/phases/tpch/denormalized/AvgAcctBal.yml

```diff
@@ -0,0 +1,44 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-query"
+Description: |
+  Run an internal TPC-H query to get the average customer account balance against the denormalized
```

Comment:
[note] IMO we should avoid benchmarking explains() because explain() enables all kinds of internal timers/metrics that actually have a fair amount of overhead (probably less than 10%, but still measurable)

Since these queries are all $groups, they return a fairly small result set, and we could get away with just using a large batch size.

Either way, it looks like the existing tests do use explain() and you were following the same model. Given this point is not specific to your new tests I see no reason to hold up the PR on this